copylot/hardware/orca_camera/camera.py

```diff
@@ -1,5 +1,40 @@
 from copylot.hardware.orca_camera.dcam import Dcamapi, Dcam
 
+import cv2
+import numpy
+from copylot.hardware.orca_camera.dcamapi4 import DCAM_IDPROP
+
+
+def dcamtest_show_framedata(data, windowtitle, iShown):
+    """
+    Show numpy buffer as an image
+
+    Arg1:   NumPy array
```

Comment:
We use NumPy style docstrings @xiyuyi , you can set your pycharm to generate NumPy style docstrings and if you like to read more about it see this: https://numpydoc.readthedocs.io/en/latest/format.html